tests/integration/cockroachdb_test.go

```diff
@@ -40,7 +40,7 @@ import (
 //   - 25% cache, 25% maxSQLMemory
 // ************************************************
 func TestCockroachDBSuite(t *testing.T) {
-	if installer.SkipTestSuite(installer.CockroachDBTestSuite) {
+	if installer.SkipTestSuite(installer.CockroachDBTestSuite) || installer.RunOnlyCephSuite() {
```

Comment:
The `installer.SkipTestSuite()` should already handle skipping the tests if only ceph tests are desired. You shouldn't need a helper for `RunOnlyCephSuite()` for all the other providers.

Reply:
Done.